sdk/src/main/java/io/opentelemetry/sdk/trace/RandomIdsGenerator.java

```diff
@@ -22,14 +22,12 @@
 
 final class RandomIdsGenerator implements IdsGenerator {
   private static final long INVALID_ID = 0;
+  private static final long MIN_VALID_ID = 1;
 
   @Override
   public SpanId generateSpanId() {
-    long id;
-    ThreadLocalRandom random = ThreadLocalRandom.current();
-    do {
-      id = random.nextLong();
-    } while (id == INVALID_ID);
+    long id = ThreadLocalRandom.current()
+            .nextLong(MIN_VALID_ID, Long.MAX_VALUE + 1);;
```

Comment:
extra semi-colon here.  Also, why `Long.MAX_VALUE + 1` ?

Reply:
The upper bound is exclusive
https://docs.oracle.com/javase/8/docs/api/java/util/stream/LongStream.html#range-long-long-